error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/ReactorRules.java

```diff
@@ -541,6 +541,284 @@ Mono<T> after(Mono<T> mono) {
     }
   }
 
+  /**
+   * Don't unnecessarily transform a {@link Flux#using(Callable, Function)} to a mono, instead use
+   * the equivalent API provided by {@link Mono}.
```

Comment:
Here and below, technically it is not strictly equivalent, unless we return in the supplier's `Function` a `Mono` in both the before and after templates.